yuca/zoo/neuroo_bicaudatus_ignis000.csv

```diff
@@ -1,17 +1,5 @@
-ca_config,nca_clone_tanh_o_bicaudatus_ignis_good.npy
+ca_config,neurorbium_bicaudatus_a.npy
 commit_hash,88fddff18e7428869c02c48b42496a7df6e3282b
-notes, Cloned from the continuous ca o_bicaudatus_ignis.npy
- NCA description: NCA(
```

Comment:
Including the instantiation kwargs make the extra notes on model structure unnecessary.